pkg/querier/querier.go

```diff
@@ -54,6 +54,7 @@ type Config struct {
 	PreferStreamingChunks                      bool   `yaml:"prefer_streaming_chunks" category:"experimental"`
 	StreamingChunksPerIngesterSeriesBufferSize uint64 `yaml:"streaming_chunks_per_ingester_series_buffer_size" category:"experimental"`
 	MinimizeIngesterRequests                   bool   `yaml:"minimize_ingester_requests" category:"experimental"`
+	QueryOnlyActiveIngesters                   bool   `yaml:"query_only_active_ingesters" category:"experimental"`
```

Comment:
Do we need a config option? I can understand you may want to have a quick way to disable it in case something doesn't work as expected, but I don't think the plan is to make it configurable. The change you're proposing (after fixing the "no extend replication set") is the behaviour we want and shouldn't be configurable.

Reply:
Makes sense, I've removed it in 92c096a